docs/reference/slate/leaf.md

```diff
@@ -0,0 +1,99 @@
+# `Leaf`
+
+```js
+import { Leaf } from 'slate'
+```
+
+A leaf node in a Slate [`Document`](./document.md). Leaf nodes are always direct children of a [`Text`](./text.md) node and contains the textual content with associated [`Marks`](./mark.md).
+
+## Properties
+
+```js
+Leaf({
+  text: string,
+  marks: Immutable.Set<Mark>
+})
+```
+
+### `text`
+
+A string that makes up the leaf node's content.
+
+### `marks`
+
+An immutable set of [`Marks`](./mark.md)
+
+### `object`
+
+`String`
+
+An immutable string value of `'leaf'`.
+
+## Computed Properties
+
+## Static Methods
+
+### `Leaf.create`
+
+`Leaf.create(properties: Object) => Leaf`
+
+Create a leaf from a plain Javascript object of `properties`.
+
+### `Leaf.fromJSON`
+
+`Text.fromJSON(object: Object) => Text`
+
+Create a leaf from a JSON `object`.
+
+### `Text.isLeaf`
+
+`Text.isLeaf(maybeLeaf: Any) => Boolean`
+
+Returns a boolean if the passed in argument is a `Leaf`.
+
+### `Text.splitLeaves`
+
+`Text.splitLeaves(leaves: List<Leaf>, offset: number) => List<Leaf>[]`
```

Comment:
I dont think this makes sense here? Either it should be `Leaf.` or it should live in `text.md`.